### ISSUE TITLE
Add Hypernova as alternative SSR service

### ISSUE THREAD

**Issue by marteinn** [Frojd/django-react-templatetags#49]
The work to include Hypernova is ongoing, this issue precedes https://github.com/Frojd/django-react-templatetags/issues/40 and is meant to be a more transparent list of what needs to be done before we can release it.

- [x] Allow more complex data to be returned from SSR service (Most importantly so we can pass Hypernova id and key from service). Ref: https://github.com/Frojd/django-react-templatetags/commit/a4c18f21440d87b86a6239902d726b2d5e331215
- [x] Include Hypernova id and key as return values from SSR service (Parse out Hypernova id and key from response html, additional SSR values are passed as params in the response dict). Ref: https://github.com/Frojd/django-react-templatetags/commit/ace55f4d7f7de5677711eb14fb7409d7405c32b6
- [x] Add extra data from SSR back into the component object stored in queue (So we can, if we wish, use the Hypernova id/key in the react_print template). Ref: https://github.com/Frojd/django-react-templatetags/commit/34e13efef6c1b280e685551e3c18ef223d40d8bc
- [x] Create tag attribute `no_placeholder` that disables the default DRTT wrapper and only uses the return value from SSR (since Hypernova includes its own div wrapper, our wrapper is redundant, depending on client side initialization method of course). Ref: https://github.com/Frojd/django-react-templatetags/commit/e41208b3e71622dd9a190771088803edc63c9ea3
- [x] Add Hypernova support in the regular `react_print.html` partial. Ref: https://github.com/Frojd/django-react-templatetags/commit/0c8c890fb840df11bf4b61a9e2e1ab70a35c446c
- [x] Create a example project with Hypernova with DRTT to show implementation. Completed: https://github.com/marteinn/django-react-polls-with-hypernova-examples
- [x] (Nice to have) Measure code coverage
- [ ] (Nice to have) Look into the double serialization of prop values in the Hypernova -> SSR bridge

**Comment by marteinn** [comment]
Version https://github.com/Frojd/django-react-templatetags/releases/tag/v6.0.0 is now officially out. Thanks for all the help and if anyone sees a bug please file a issue. Closing.
